apps/omg_db/mix.exs

```diff
@@ -31,17 +31,23 @@ defmodule OMG.DB.MixProject do
 
   defp deps do
     [
-      {:rocksdb, "~> 1.2"},
       {:exleveldb, "~> 0.11"},
+      {:omg_status, in_umbrella: true},
```

Comment:
let's keep `inumbrella` ones at the bottom

Reply:
these are split between what's used in test and what in a release...